client/tests/integration/triggers/by_call_trigger.rs

```diff
@@ -299,18 +300,25 @@ fn unregister_trigger() -> Result<()> {
 /// Despite this simplification this test should really check
 /// if we have the ability to pass a base64-encoded WASM trigger in the genesis.
 #[test]
-// TODO: Same suggestion here as in #2641.
-// Let's use feature instead of ignore.
-// This feature will be activated by the build-script on nightly builds/
-#[ignore = "Only on nightly"]
 fn trigger_in_genesis_using_base64() -> Result<()> {
-    // Reading wasm smartcontract
-    let wasm = std::fs::read(concat!(
-        env!("OUT_DIR"),
-        "/wasm32-unknown-unknown/release/mint_rose.wasm"
-    ))
-    .wrap_err("Can't read smartcontract")?;
-    println!("wasm size is {} bytes", wasm.len());
+    // Building wasm trigger
+
+    let temp_out_dir =
+        tempfile::tempdir().wrap_err("Failed to create temporary output directory")?;
+
+    info!("Building trigger");
+    let wasm = iroha_wasm_builder::Builder::new("tests/integration/smartcontracts/mint_rose")
+        .out_dir(temp_out_dir.path())
+        .build()?
+        .optimize()?
```

Comment:
why is `optimize` separate from `build`? Is there a case where we want only to build and not to optimize? Also it's possible to forget to optimize this way

Reply:
Well, I'm not sure. Just following the principles **The least surprise** and **One function does one thing**. Also maybe `optimize()` can give an error when `build()` is okay.

Reply:
Let's wait for other opinions.

@appetrosyan , @Erigara

Reply:
> In the real world, if you want to build something with Rust, you need to opt-into optimisations

Yes, but it's like parameter to build not a separate step. 

Can we clearly separate process of building and optimizing in case of wasm?

Reply:
> Can we clearly separate process of building and optimizing in case of wasm?

Yes! By *build()* I mean running `cargo build`. By `optimize()` I mean running `wasm-opt` to opmtize size

Reply:
Okay, looks like we decided to keep it separated. I'm closing this